docs/DeveloperGuide.md

```diff
@@ -672,6 +672,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### **1. More Specific Success Message for Company**
 
+**Feature Flaw in Current Implementation**
+
+Currently, the success message for view, edit, remark, unremark, and add commands only displays the company name.
+However, the duplicate check implemented in LinkMeIn uses 3 criteria, company name, role, and deadline. As such, there
+can be more than one entry with the same company name. The user might want to know the role and deadline to
+differentiate between applications with the same company name. As such, with the current implementation of the success
+message, it can be confusing for the user as to which company specifically has been modified.
+
+**Proposed Enhancement**
+
+Instead of displaying only the company name, the success message will display the company name, role and deadline. This
+is the implementation that is done for the existing `delete` command. As such, a method was already created in
+the `Messages` class called `getCompanyInfo` where the company name, role and deadline from the company object will be
+returned as a string.
+
+The following implementation will be adopted instead:
+`return new CommandResult(String.format(MESSAGE_SUCCESS, Messages.getCompanyInfo(toAdd)));`
+
+**Examples**
+
+- **view**: `Viewing Company: COMPANY_NAME (ROLE, DEADLINE)`
+- **edit**: `COMPANY_NAME (ROLE, DEADLINE) company edited.`
+- **remark**: `Added remark to Company: COMPANY_NAME (ROLE, DEADLINE)`
+- **unremark**: `Removed remark from Company: COMPANY_NAME (ROLE, DEADLINE)`
+- **add**: `New company added: COMPANY_NAME (ROLE, DEADLINE)`
```

Comment:
- **view**: `Viewing company: COMPANY_NAME (ROLE, DEADLINE)`
- **edit**: `COMPANY_NAME (ROLE, DEADLINE) company edited.`
- **remark**: `Added remark to company: COMPANY_NAME (ROLE, DEADLINE)`
- **unremark**: `Removed remark from company: COMPANY_NAME (ROLE, DEADLINE)`
- **add**: `New company added: COMPANY_NAME (ROLE, DEADLINE)`


change the "Company" to small letters for standardisation